### PR TITLE
[Fix-16290] Filter the NaN of system_cpu_usage/process_cpu_usage

### LINKED ISSUE #16290: [Bug]  system_cpu_usage/process_cpu_usage will be NaN
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

This will cause the master/worker overload, we should filter these value.

### What you expected to happen

xx

### How to reproduce

xx

### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by ruanwenjun**
## Purpose of the pull request

close https://github.com/apache/dolphinscheduler/issues/16290

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

## Pull Request Notice
[Pull Request Notice](https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/contribute/join/pull-request.md)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: dolphinscheduler-meter/src/main/java/org/apache/dolphinscheduler/meter/metrics/DefaultMetricsProvider.java [hunk 1]
```diff
@@ -33,6 +33,9 @@ public DefaultMetricsProvider(MeterRegistry meterRegistry) {
 
     private long lastRefreshTime = 0;
 
+    private double lastSystemCpuUsage = 1.0d;
+    private double lastProcessCpuUsage = 1.0d;
```
Comment (Gallardot) [inline dolphinscheduler-meter/src/main/java/org/apache/dolphinscheduler/meter/metrics/DefaultMetricsProvider.java]: Maybe 0.0d is a better default?

Comment (ruanwenjun) [reply dolphinscheduler-meter/src/main/java/org/apache/dolphinscheduler/meter/metrics/DefaultMetricsProvider.java]: Agree with you, updated.

**Review by SbloodyS — APPROVED** [review]
LGTM

**Review by Gallardot — APPROVED** [review]
LGTM

**Pull request merged**